src/containers/OwnerView.js

```diff
@@ -18,7 +18,7 @@ import PartyChip from '../components/PartyChip';
 
 function OwnerView({ owner }) {
   const [characters, updateCharacters] = useState([]);
-  const [parties, updateParties] = useState({});
+  const [parties, updateParties] = useState([]);
```

Comment:
This is a 2D array where the first index of every inner array is the name of the party.

Is there a reason you didn't want to do an array of objects?

```js
[
  {
    name: "New Beginnings",
    members: []
  }
]
```